en/lessons/basics/pipe-operator.md

```diff
@@ -1,5 +1,5 @@
 ---
-version: 1.0.0
+version: 1.0.1
```

Comment:
Currently in case when the translation's version is higher than the original lesson's version is considered an error. I am not sure whether this kind of behaviour is correct.
/cc @elixirschool/developers

Reply:
@nscyclone that's an interesting point.  I would imagine only the patch version would increment out of sync with the english lesson?

Reply:
@doomspork I think so. When there is an update to an actual translation we just bump the patch version. Should we remove the warning for patch versions mismatch?

Reply:
I think that is a reasonable solution @nscyclone 👍